engines/decidim_hospitalet-surveys/app/views/devise/mailer/invite_user.html.erb

```diff
@@ -0,0 +1,13 @@
+<p><%= t("devise.mailer.invitation_instructions.hello", email: @resource.name) %></p>
+
+<p>
+  <%= t("devise.mailer.invitation_instructions.someone_invited_you", application: @resource.organization.name) %>
```

Comment:
Maybe we should tweak a little bit this email, it's not just that someone has invited you. It's something like your survey has made into the system and if you want you can continue participating at the website. /cc @carolromero